src/headless/converse-chat.js

```diff
@@ -1100,7 +1100,17 @@ converse.plugins.add('converse-chat', {
                     return;
                 }
                 if (utils.isNewMessage(message) && this.isHidden()) {
-                    this.save({'num_unread': this.get('num_unread') + 1});
+                    let first_unread = this.get('first_unread');
```

Comment:
`first_unread` doesn't give you any indication what the value is. Given that it's an ID, I think this key should be named `first_unread_id`.

Reply:
ok. It will be renamed